components/__design__/hex-logos.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const HexLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <div>
+          <Image
+            src={`/static/images/logos/hex-white.svg`}
+            alt="Node.js"
+            width={64}
+            height={64}
+          />
+        </div>
+        <div>
+          <Image
+            src={`/static/images/logos/hex-green.svg`}
+            alt="Node.js"
+            width={64}
+            height={64}
+          />
+        </div>
```

Comment:
Let's not add these because the files haven't been added.

Reply:
Done! Please review again! Thank you😊 @HinataKah0